package.json

```diff
@@ -89,6 +89,7 @@
     "module-details-from-path": "^1.0.3",
     "node-abort-controller": "^3.0.1",
     "opentracing": ">=0.12.1",
+    "pako": "^2.1.0",
```

Comment:
Does this just do gzip encoding? Does it do so differently than the one [built into Node.js](https://nodejs.org/api/zlib.html)?